src/stores/businessStore.ts

```diff
@@ -277,17 +283,16 @@ export const useBusinessStore = defineStore('business', {
       return new Promise((resolve, reject) => {
         const businessId = sessionStorage.getItem('BUSINESS_ID')
 
-        // if there is no business id, return error
+        // If there is no business id, return an error.
         if (!businessId) {
           reject(new Error('Missing business id'))
           return
         }
-
         LegalServices.fetchBusiness(businessId)
```

Comment:
The whitespace above was to separate the error checking from the rest of the code. Please restore it.